src/main/java/address/MainApp.java

```diff
@@ -41,34 +41,38 @@
     protected UpdateManager updateManager;
     protected MainController mainController;
     protected KeyBindingsManager keyBindingsManager;
+    protected Config config;
 
     public MainApp() {}
 
     @Override
     public void init() throws Exception {
         logger.info("Initializing app ...");
         super.init();
+        initComponents();
         initConfig();
-        Config.setConfig(Config.getConfig());
         initPrefs();
         BrowserManager.initializeBrowser();
         //TODO: should this be here? looks out of place
-        initComponents();
```

Comment:
There's a problem with this order.
- Initializing the components also means initializing the storage manager, which is responsible for reading from the config file. i.e. the config will get overwritten by components initialization in TestApp.

Either way, there's some weird Exception being thrown in the start method at the moment. Will need to figure that out first.